resources/lib/PayoneApi/Request/GenericPayment/AmazonPayConfirmOrderReferenceRequest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace PayoneApi\Request\GenericPayment;
+
+use PayoneApi\Request\ClearingTypes;
+use PayoneApi\Request\Parts\Config;
+use PayoneApi\Request\Parts\SystemInfo;
+use PayoneApi\Request\Types;
+
+class AmazonPayConfirmOrderReferenceRequest
```

Comment:
Hier auch wieder Variablen nicht in camelCase und hier jetzt einheitlich keine Typ Definitionen mehr.
Können wir uns da für entweder/oder entscheiden? 😄

Reply:
Siehe oben bzgl. `json_serialize`.